### PR TITLE
Update Security_Elasticsearch.md

### PR DESCRIPTION
**Pull request by daudahabbasi**

### CODE REVIEW

**Comment by daudahabbasi** [comment]
Adding @jens-gellynck also for review.

File: user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Database_security/Security_Elasticsearch.md [hunk 1]
```diff
@@ -77,6 +77,17 @@ To configure TLS encryption for client-server communication:
    xpack.security.http.ssl.keystore.path: path/to/your/certificate
    xpack.security.http.ssl.truststore.path: path/to/your/certificate
    ```
+   Add the following lines to *elasticsearch.yml* file, if you are using `.PEM` certificate generated using `penssl utility` .
```
Comment (jens-gellynck) [inline user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Database_security/Security_Elasticsearch.md:80]: Should be 'openssl' utility

File: user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Database_security/Security_Elasticsearch.md [hunk 2]
```diff
@@ -77,6 +77,17 @@ To configure TLS encryption for client-server communication:
...
+   
+   ```
+    xpack.security.http.ssl.enabled: true
+    xpack.security.http.ssl.verification_mode: full
+    xpack.security.http.ssl.key: path/to/your/PrivateKey/used/to/generatet/the/certificate
+    xpack.security.http.ssl.certificate: path/to/your/certificate
+    xpack.security.http.ssl.certificate_authorities: path/to/certificate/authority (if you are using self-signed
```
Comment (jens-gellynck) [inline user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Database_security/Security_Elasticsearch.md:87]: maybe add the last part as a comment (prefixed by '#') so the yaml stays valid if someone just copy pastes this

File: user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Database_security/Security_Elasticsearch.md [hunk 3]
```diff
@@ -92,6 +103,8 @@ To configure TLS encryption for client-server communication:
 1. Add the full *https://* URL (including the port) in the \<DBServer> element in the *DB.xml* file. For example:
    `<DBServer>https://elastic.dataminer:9200</DBServer>`
 
+1. Add the certificates on each Dataminer agent under `Trusted root certification authorities`.
```
Comment (jens-gellynck) [inline user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Database_security/Security_Elasticsearch.md:106]: Import the certificate in the 'Trusted Root Certification Authorities' store of the Windows Certificate store.

File: user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Database_security/Security_Elasticsearch.md [hunk 4]
```diff
@@ -113,6 +126,16 @@ To configure TLS encryption for inter-node communication:
    xpack.security.transport.ssl.keystore.path: elastic-certificates.p12 
    xpack.security.transport.ssl.truststore.path: elastic-certificates.p12
    ```
+    Add the following lines to *elasticsearch.yml* file, if you are using `.PEM` certificate generated using `penssl utility` .
```
Comment (jens-gellynck) [inline user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Database_security/Security_Elasticsearch.md:129]: *openssl

File: user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Database_security/Security_Elasticsearch.md [hunk 5]
```diff
@@ -113,6 +126,16 @@ To configure TLS encryption for inter-node communication:
...
+   
+   ```
+    xpack.security.transport.ssl.enabled: true
+    xpack.security.transport.ssl.verification_mode: full 
+    xpack.security.transport.ssl.key: path/to/your/PrivateKey/used/to/generatet/the/certificate
+    xpack.security.transport.ssl.certificate: path/to/your/certificate
+    xpack.security.transport.ssl.certificate_authorities: path/to/certificate/authority (if you are using self-signed
```
Comment (jens-gellynck) [inline user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Database_security/Security_Elasticsearch.md:136]: same

**Review by jens-gellynck — COMMENTED** [review]
Some small feedback

File: user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Database_security/Security_Elasticsearch.md [hunk 6]
```diff
@@ -126,15 +127,16 @@ To configure TLS encryption for inter-node communication:
    xpack.security.transport.ssl.keystore.path: elastic-certificates.p12 
    xpack.security.transport.ssl.truststore.path: elastic-certificates.p12
    ```
-    Add the following lines to *elasticsearch.yml* file, if you are using `.PEM` certificate generated using `penssl utility` .
+    Add the following lines to *elasticsearch.yml* file, if you are using `.PEM` certificate generated using `penssl` utility.
```
Comment (jens-gellynck) [inline user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Database_security/Security_Elasticsearch.md:130]: openssl?

**Review by jens-gellynck — CHANGES_REQUESTED** [review]
replace penssl with 'openssl'

**Comment by MariekeGO** [comment]
@jens-gellynck I've made a few more small changes. Could you confirm if this is now OK to merge?

**Review by jens-gellynck — APPROVED** [review]
LGTM

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**